k의 개수

```diff
@@ -0,0 +1,14 @@
+int solution(int i, int j, int k) {
+        int answer = 0;
+        for (int t = i; t <= j; t++ ){
+            StringBuilder sb = new StringBuilder();
+            sb.append(t);
+
+            for (int r = 0; r < sb.length(); r++) {
+                if ((sb.charAt(r) - '0') == k) {
+                    answer++;
+                }
+            }
+        }
+        return answer;
+    }
```

Comment:
Your solution is a success. 👏
<img width="588" alt="스크린샷 2023-02-24 오후 9 23 51" src="https://user-images.githubusercontent.com/4418255/221178880-a85310e0-89eb-44b0-a158-1bfa96762ffa.png">